README.md

```diff
@@ -10,12 +10,25 @@ See the [documentation](https://algolia.com/doc/tools/cli/) for setup and usage
 
 ## Installation
 
+### From repository
+
+A `Makefile` is available to help installing and building the CLI.
+
+```bash
+make install
+```
+
 ### MacOS
 
 `algolia` is available on Homebrew and as a downloadable binary from the [releases page](https://github.com/algolia/cli/releases).
 
 ```bash
 brew tap algolia/algolia-cli
+```
+
+then
+
+```bash
```

Comment:
I think I prefer the readme without this gap, what do you think Loic? More concise.

I say this because I'm about to add some content to the readme, but I still want it compact and neat as possible.

We could even collapse into one line:
`brew tap algolia/algolia-cli && brew install algolia`

Reply:
Sure you're right, it's better! 👍